Makefile

```diff
@@ -108,6 +108,9 @@ build-daemon:
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run cmd/operator/main.go
 
+ifndef IGNORE-NOT-FOUND
+  IGNORE-NOT-FOUND = false
```

Comment:
For the variable name, consider using snake case ` IGNORE_NOT_FOUND`

Reply:
I'd like to keep this as close to OSDK generated code as much as possible.